projects/observability/src/shared/dashboard/data/graphql/explore-error-percentage/explore-error-percentage-data-source.model.ts

```diff
@@ -17,9 +17,9 @@ import {
 } from '../../../../graphql/request/handlers/explore/explore-graphql-query-handler.service';
 
 @Model({
-  type: 'api-error-percentage-data-source'
+  type: 'explore-error-percentage-data-source'
```

Comment:
Going forward, we should try to keep the implementation out of the class/model names - the fact that this uses the explore API isn't relevant, it's just an `error-percentage-data-source`

Reply:
this is intended as a customizable error percentage data source that works on explore query, hence the name. so that we can avoid defining these again for similar use cases.

Reply:
but if we're customizing the numerator and the denominator, isn't it just a percentage data source? there's nothing tied to errors in it, and how it resolves the data isn't relevant to someone selecting a model

Reply:
this one can be just a percentage data source model for explore queries that need no filters to be applied. The name is just left over from the unused model that we revamped. will fix it.

Reply:
On revisiting, I think for our current use cases, we can use the other data source from the above linked PR for this purpose as well. 
That also _percentages_ values against the total call count, which is what we are trying to do here as well.

Reply:
I'm imagining something of the pseudo form below. It's more verbose, but I think handles all the cases we need. The only gotcha right now is an aggregation data source doesn't support context, but we could add that if need be (not sure, do we have a use case there?)
```
{
  type: percentage-data-source,
  context: "API"
  numerator: {
    type: metric-aggregation-data-source, // These can already apply filters
    metric: 'foo',
    aggregation:  SUM
  },
  denominator: {
    type: metric-aggregation-data-source,
    metric: 'bar',
    aggregation:  SUM
  }
}
```

Reply:
Yes, this is the obvious solution. I thought the reason why we didn't do this in UI already is because we had plans to move percentage calculation to backend. Any ideas about that?

Reply:
I'd guess they're not on the short term road map, and likely wouldn't support the type of flexibility we'd need here anyway - things like apply filters to just the numerator/denominator, or using different contexts for both.

Reply:
I'll create this data source for explore queries next.